git/03.commit-messages.md

```diff
@@ -55,6 +55,12 @@ Return 404 on permissions API when no access [TR #45] (GH #4)
 If a user cannot access a study at all we want to be able communicate that to API clients, thus we will return a 404.
 ```
 
+Note: when using abbreviate references to a card or PR that will have
+a limited lifespan if the board or repo host is changed.  If there is
+critical information related to the commit, copy it into the commit message body
+and consider using using fully qualified URLs if you want to ensure link will
+work in the future.
```

Comment:
@calebwoods I think the second sentence here makes sense, but I'm struggling to understand what you mean by the first one. :-) Can you clean it up a bit?